addOns/network/src/main/java/org/zaproxy/addon/network/ExtensionNetwork.java

```diff
@@ -1478,6 +1560,65 @@ String getProxyPacContent(String hostname) {
         return sb.toString();
     }
 
+    private AbstractPanel getRateLimitStatusPanel() {
+        if (rateLimitStatusPanel == null) {
+            rateLimiterTableModel = new RateLimiterTableModel();
+            rateLimiter.setObserver(
+                    limiter -> {
+                        if ((System.currentTimeMillis() - rateLimiterLastUpdate.get()) > 1000) {
+                            rateLimiterLastUpdate.set(System.currentTimeMillis());
+                            SwingUtilities.invokeLater(
+                                    () -> {
+                                        rateLimiterTableModel.update(limiter);
+                                    });
+                        }
+                    });
+
+            ZapTable limiterTable = new ZapTable(rateLimiterTableModel);
+            limiterTable.setColumnSelectionAllowed(false);
+            limiterTable.setCellSelectionEnabled(false);
+            limiterTable.setRowSelectionAllowed(false);
+            limiterTable.setAutoCreateRowSorter(true);
+            limiterTable.setAutoCreateColumnsFromModel(true);
+            limiterTable.setColumnControlVisible(true);
+            limiterTable.setDoubleBuffered(true);
+            limiterTable.setDefaultRenderer(
+                    Date.class, new DefaultTableRenderer(new DateFormatStringValue()));
+
+            JScrollPane scroller = new JScrollPane();
+            scroller.setName("RateLimiterPane");
+            scroller.setViewportView(limiterTable);
+            limiterTable.packAll();
+
+            rateLimitStatusPanel = new AbstractPanel();
+            rateLimitStatusPanel.setLayout(new BorderLayout());
+            rateLimitStatusPanel.setName(
+                    Constant.messages.getString("network.ratelimit.status.title"));
+            rateLimitStatusPanel.setIcon(DisplayUtils.getScaledIcon(RATE_LIMIT_ICON));
```

Comment:
I scaled this one, I believe that was the right move :wink:

Reply:
The icon is being scaled by core.

Reply:
Oh so I don't need to do either of them here?

Reply:
This one is not scaled, the context menu needs to be.

Reply:
Okay thanks for clarifying.